Diccionario de datos

Skjaldborg95 opened the pull request:
Diccionario de datos con las 25 tablas del módelo de datos.

Skjaldborg95 committed:
Diccionario de datos

3/25

Skjaldborg95 committed:
Diccionario de datos

Completado.

baezahuertaelias commented:
- La tabla ENTRADA_FICHA, no será mejor dejarla un poco más específica como por ejemplo OBS_FICHA?
- La tabla PERS_MEDICO, no será mejor dejarla un poco más específica como por ejemplo TURNO_PMEDICO?
- La tabla PRESTACION, detallar que tipo de prestación.
- La tabla TIPO_EQUIPO, detallar que tipo de equipo.
- El detalle de la tabla ASEGURADORA, yo lo dejaría como "Tabla que registra el nombre de la aseguradora"
- El campo OBSERVACIONES de la tabla ATENCION_AGEN, le daría 80 caracteres como mínimo, hice una prueba y ocupé 76 caracteres.
- En el campo RUT de la tabla PACIENTE, con que el largo del varchar sea de 13 (dando el ejemplo de un rut XX.XXX.XXX-X) es más que suficiente
- El campo COMENTARIO de la tabla RES_ATENCION, le daría 140 caracteres como mínimo, hice una prueba y ocupé 135 caracteres.
- Algo que no es muy relevante pero en la pagina 6 pierdes mucho espacio de hoja en blanco.
- En la tabla DEVOLUCION, pensando en los BPMN, cuando se genera una devolución, existe la posiblidad de que haya que crear una 'nota de crédito', eso según yo debería ser considerado en esa tabla

psnull commented:
Resueltos comentarios de Elías, pero queda pendiente lo siguiente:
- [x] La tabla PRESTACION, detallar que tipo de prestación.
- [x] La tabla TIPO_EQUIPO, detallar que tipo de equipo.
- [x] El detalle de la tabla ASEGURADORA, yo lo dejaría como "Tabla que registra el nombre de la aseguradora"
- [x] Algo que no es muy relevante pero en la pagina 6 pierdes mucho espacio de hoja en blanco.

Skjaldborg95 committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into diccionario-de-datos

Skjaldborg95 committed:
Diccionario de datos

La tabla PRESTACION, detallar que tipo de prestación.
La tabla TIPO_EQUIPO, detallar que tipo de equipo.
El detalle de la tabla ASEGURADORA, yo lo dejaría como "Tabla que
registra el nombre de la aseguradora".
Algo que no es muy relevante pero en la pagina 6 pierdes mucho espacio
de hoja en blanco.

Skjaldborg95 commented:
Correcciones hechas.
